### PR TITLE
Remove spritecomponent redefinitions from species

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
## About the PR 
Felinid, Moth, and Oni didn't seem to need the extra redefinition of the entire sprite component, so I removed it, instead using the parent type.

<details><summary>picture just in case</summary>
<p>

image

</p>
</details>

### CODE REVIEW

**Comment by FluffiestFloof** [comment]
This results in moth arms and legs markings appearing above clothing.
ss+(2023-05-03+at+02 10 53)

**Comment by Elijahrane** [comment]
might be fine (?) for felinid and oni
it will break moths though

**Comment by DEATHB4DEFEAT** [comment]
> This results in moth arms and legs markings appearing above clothing.
> ss+(2023-05-03+at+02 10 53)

looks pretty cool tbh, but I'll figure out what's wrong

**Comment by FluffiestFloof** [comment]
> looks pretty cool tbh, but I'll figure out what's wrong

This was fixed by having this in the _type: Sprite_

https://github.com/Nyanotrasen/Nyanotrasen/blob/c882725dbfbed59bfd6a4c78a584f03cce9d2b6c/Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/moth.yml#L79-L83

Aka rendering the jumpsuit above the markings that are placed on the feet and hands.
The markings for arms/legs are placed on hand/foot due to how the layering works and how they impact both hand and arm or foot and leg.

**Comment by DEATHB4DEFEAT** [comment]
> > looks pretty cool tbh, but I'll figure out what's wrong
> 
> This was fixed by having this in the _type: Sprite_
> 
> https://github.com/Nyanotrasen/Nyanotrasen/blob/c882725dbfbed59bfd6a4c78a584f03cce9d2b6c/Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/moth.yml#L79-L83
> 
> Aka rendering the jumpsuit above the markings that are placed on the feet and hands. The markings for arms/legs are placed on hand/foot due to how the layering works and how they impact both hand and arm or foot and leg.

why isn't it like this in the base..

File: Resources/Prototypes/Entities/Mobs/Species/base.yml [hunk 1]
```diff
@@ -116,11 +116,11 @@
         sprite: Mobs/Customization/masking_helpers.rsi
         state: female_full
         visible: false
-      - map: [ "jumpsuit" ]
       - map: [ "enum.HumanoidVisualLayers.LHand" ]
       - map: [ "enum.HumanoidVisualLayers.RHand" ]
       - map: [ "enum.HumanoidVisualLayers.LFoot" ]
       - map: [ "enum.HumanoidVisualLayers.RFoot" ]
+      - map: [ "jumpsuit" ]
```
Comment (Vordenburg) [inline Resources/Prototypes/Entities/Mobs/Species/base.yml:123]: Why was this moved?

**Comment by Vordenburg** [comment]
This probably is alright for Felinid and Oni, but I'd have to do some testing to see what happens with Moth.

**Comment by FluffiestFloof** [comment]
The idea was to have the jumpsuit go above the arm markings due to those markings being on the hand layer and foot layer.
The issue, that no one noticed or complained about, is **the hands are hidden by the jumpsuit** on the side views.

Since the moths hands are so small and the wings kinda hide most of those, it's not _that_ noticeable (gloves are layered above).
On normal hands it's very visible that the hands are missing.
